kernel_tuner/util.py

```diff
@@ -102,9 +102,9 @@ class StopCriterionReached(Exception):
     "block_size_x",
     "block_size_y",
     "block_size_z",
-    "ngangs",
-    "nworkers",
-    "vlength",
+    # "ngangs",
+    # "nworkers",
+    # "vlength",
```

Comment:
Why are these lines commented?

Reply:
These block size names were added for OpenACC. The default block sizes were added automatically if the block size is not defined by the user, causing issues with these parameters outside of OpenACC. I spoke with @isazi about this, if I understood correctly these are no longer needed, hence commented pending removal.